Patterns/RequestHandler.md

```diff
@@ -0,0 +1,40 @@
+# Request Handling
+
+Request Handlers are used when our endpoint receives a request, and we need to process it:
+- Verify input query parameters or body, and map to POCO.
+- Dispatch the POCO where the logic will be conducted.
+- Receive output and form a proper HTTP Response.
+
+## Background information

```

Comment:
Remove this section as it distracts. The request handler is a simple middleware solution for function apps that do not have middleware, and ASP.NET Core should use a different approach.